constrained_ik/src/moveit_interface/cartesian_planner.cpp

```diff
@@ -38,13 +38,14 @@
 
 namespace constrained_ik
 {
-  CartesianPlanner::CartesianPlanner(const std::string &name, const std::string &group) :
-    constrained_ik::CLIKPlanningContext(name, group),
+  CartesianPlanner::CartesianPlanner(const std::string &name, const std::string &group, const std::string &ns) :
+    planning_interface::PlanningContext(name, group),
     terminate_(false),
     robot_description_("robot_description")
   {
+    resetPlannerConfiguration();
     solver_.reset(new Constrained_IK());
-    std::string constraint_param = "constrained_ik_solver/" + getGroupName() + "/constraints";
+    std::string constraint_param = ns + "/" + getGroupName() + "/constraints";
```

Comment:
You are correct. I changed it to where the node handle is passed in and used to to initialize the Constrained_IK class so everything is pulling from the same namespace when looking for parameters.